### PR TITLE
Add availability api endpoint

### PR DESCRIPTION
**Pull request by Haziba**
**Notes to tester:**
Not sure how I feel about the parameter explanations but I don't know if they work too well anyway. Might be better to take them out, add example parameter objects in each method demonstrating how to call the method. 

Not really a point about this PR but just jotting my thoughts somewhere that someone may comment on them

### CODE REVIEW

File: cronofy.php [hunk 1]
```diff
@@ -445,6 +445,33 @@ public function change_participation_status($params)
...
+          participants : Array of member arrays and the required number of people to be available from each array. REQUIRED
+                         for example: array(
+                                        array("members" => array(
+                                          array("sub" => "acc_567236000909002"),
+                                          array("sub" => "acc_678347111010113")
+                                        ), "required" => "all")
+                                      )
+          required_duration : Array detailing required period duration. REQUIRED
+                         for example: array("minutes" => 60)
+          available_periods : Array of periods within which to look for availability. REQUIED
```
Comment (gshutler) [inline cronofy.php]: `REQUIRED`

File: cronofy.php [hunk 2]
```diff
@@ -445,6 +445,33 @@ public function change_participation_status($params)
         return $this->http_post("/" . self::API_VERSION . "/calendars/" . $params["calendar_id"] . "/events/" . $params["event_uid"] . "/participation_status", $postfields);
     }
 
+    public function availability($params)
+    {
+        /*
+          participants : Array of member arrays and the required number of people to be available from each array. REQUIRED
```
Comment (gshutler) [inline cronofy.php]: Could just lift this from [the docs](https://www.cronofy.com/developers/api/alpha/#availability):

> An array of the groups of participants whose availability should be taken into account

File: cronofy.php [hunk 3]
```diff
@@ -445,6 +445,33 @@ public function change_participation_status($params)
...
+                         for example: array(
+                                        array("members" => array(
+                                          array("sub" => "acc_567236000909002"),
+                                          array("sub" => "acc_678347111010113")
+                                        ), "required" => "all")
+                                      )
+          required_duration : Array detailing required period duration. REQUIRED
```
Comment (gshutler) [inline cronofy.php]: Similarly (as this doesn't read well at all):

> Duration that an available period must last to be considered viable

File: cronofy.php (hunk 1 above)
Comment (gshutler) [inline cronofy.php]: Again, C&P from the docs:

> An array of available periods within which suitable matches may be found

**Comment by Haziba** [comment]
@gshutler :mag:

**Pull request merged**